python/chemiscope/structures/_ase.py

```diff
@@ -303,7 +303,13 @@ def _is_convertible_to_property(value):
                 np.array(value, dtype=float)
                 return True
             except Exception:
-                return False
+                try:
+                    if isinstance(np.array(value).flatten()[0], str):
+                        return True
+                    else:
+                        return False
+                except Exception:
+                    return False
```

Comment:
I'm not sure what this is trying to detect. Could you add a test for it? Also, since it seems to be testing for string compatibility, it should not be in the "everything convertible to float" else block.